tests/dataset/test_built_in_data.py

```diff
@@ -20,29 +21,29 @@ def test_populate(self):
 class TestLoomDataset(TestCase):
     def test_retina_load_train_one(self):
         dataset = scvi.data.retina(save_path="tests/data")
-        scvi.data.setup_anndata(dataset, batch_key="batch")
+        _setup_anndata(dataset, batch_key="batch")
         unsupervised_training_one_epoch(dataset)
 
     def test_pfc_starmap_load_train_one(self):
         gene_dataset = scvi.data.prefrontalcortex_starmap(save_path="tests/data")
-        scvi.data.setup_anndata(gene_dataset)
+        _setup_anndata(gene_dataset)
         unsupervised_training_one_epoch(gene_dataset)
 
     def test_fc_dropseq_load_train_one(self):
         gene_dataset = scvi.data.frontalcortex_dropseq(save_path="tests/data")
-        scvi.data.setup_anndata(gene_dataset)
+        _setup_anndata(gene_dataset)
         unsupervised_training_one_epoch(gene_dataset)
 
     def test_smfish_load_train_one(self):
         gene_dataset = scvi.data.smfish(save_path="tests/data")
-        scvi.data.setup_anndata(gene_dataset)
+        _setup_anndata(gene_dataset)
         unsupervised_training_one_epoch(gene_dataset)
 
 
 class TestSeqfishDataset(TestCase):
     def test_populate(self):
         dataset = scvi.data.seqfish(save_path="tests/data")
-        scvi.data.setup_anndata(dataset)
+        _setup_anndata(dataset)
```

Comment:
we can probably add setup anndata to the `unsupervised_training_one_epoch` function? and then remove all these lines